src/Emanote/Pandoc/BuiltinFiltersSpec.hs

```diff
@@ -0,0 +1,48 @@
+module Emanote.Pandoc.BuiltinFiltersSpec where
+
+import Emanote.Pandoc.BuiltinFilters
+import Emanote.Pandoc.Markdown.Parser (parseMarkdown)
+import Hedgehog
+import Relude
+import Test.Hspec
+import Test.Hspec.Hedgehog (hedgehog)
+import Text.Pandoc.Definition (Inline (..))
+import Text.Pandoc.Walk (query)
+
+spec :: Spec
+spec =
+  do
+    describe "setExternalLinkicon" $ do
+      let run =
```

Comment:
Why is this called "run"? What does this function do? What does it return? You want to clarify all that in some fashion.

Reply:
I hope it looks better now.

Reply:
It is :-) I've also refactored it a bit more to satisfy my subjective view on clarity.